tests/base_tests.py

```diff
@@ -1,14 +1,15 @@
 from unittest import TestCase
 from loguru import logger
 from uuid import uuid4
+from subprocess import Popen, PIPE, STDOUT
 
 
 class BaseTests(TestCase):
     LOGGER = logger
 
     def setUp(self):
-        print('\t')
-        self.info('Test case : {}'.format(self._testMethodName))
+        print("\t")
+        self.info("Test case : {}".format(self._testMethodName))
```

Comment:
what's self.info? and what about j.logger? we can even create custom format for tests if needed.